crypto/nrf_cc310_mbedcrypto/include/mbedtls/dhm_alt.h

```diff
@@ -73,6 +59,5 @@ mbedtls_dhm_context;
 }
 #endif
 
-#endif  /* MBEDTLS_DHM_ALT */
-
-#endif  /* MBEDTLS_DHM_ALT_H */
+#endif  /* MBEDTLS_DHM_ALT  - use alternative code */
+#endif  /* MBEDTLS_DHM_ALT_H  - include only once  */
```

Comment:
You don't have to explain what a header guard is.

Reply:
Will fix